static/app/components/replays/breadcrumbs/replayTimelineTooltip.tsx

```diff
@@ -21,51 +21,48 @@ type Props = {
 };
 
 function TimelineTooltip({enabled = true, labelText}: Props) {
-  const rafIdRef = useRef<number | null>(null);
   const labelRef = useRef<HTMLDivElement>(null);
+  const portalRef = useRef<HTMLElement | null>(null);
+  const [position, setPosition] = useState({x: 100, y: FIXED_Y_POSITION});
+  const [isVisible, setIsVisible] = useState(true);
 
-  const [isVisible, setIsVisible] = useState(false);
-  const [offsetLeft, setOffsetLeft] = useState(0);
-
-  const portal = document.getElementById('replay-timeline-player');
+  useEffect(() => {
+    portalRef.current = document.getElementById('replay-timeline-player');
+  }, []);
```

Comment:
Hmm I'd probably try to avoid `useEffect` if possible.

Could we check if `portalRef.current` exists inside of the mouseMove handler, and if not, set it?